test_db.py

```diff
@@ -0,0 +1,231 @@
+import unittest
+import uuid
+from PIL import Image
+import io
+import base64
+import db.queries.user as user
+import db.queries.seed as seed
+import db.queries.picture as picture
+import metadata.picture_set as picture_set_data
+import metadata.picture as picture_data
+import db.__init__ as db
+
+"""
+This is a test script for the database packages. 
+It tests the functions in the user, seed and picture modules.
+"""
```

Comment:
Move your docstring on top of the import as they usually are written before any code line